addons/web_unsplash/static/src/components/media_dialog/image_selector.xml

```diff
@@ -10,29 +10,27 @@
 
 <t t-name="web_unsplash.UnsplashCredentials" owl="1">
     <div class="d-flex align-items-center flex-wrap">
-        <a href="https://www.odoo.com/documentation/master/applications/websites/website/optimize/unsplash.html#generate-an-unsplash-access-key"
-           class="mx-2" target="_blank">Get an Access key</a>
+        <a href="https://www.odoo.com/documentation/16.0/applications/websites/website/optimize/unsplash.html#generate-an-unsplash-access-key"
+           class="me-2" target="_blank">Get an Access key</a>
```

Comment:
Could have been 1 to have a natural word spacing instead of this weird double spacing but it's fine 
Edit: Since there is another change needed, and if you agree, maybe quickly change this one (and ms-2 below) too)